pkg/forklift-api/webhooks/validating-webhook/admitters/secret-admitter.go

```diff
@@ -227,3 +233,43 @@ func (admitter *SecretAdmitter) testConnectionToHost(hostName string) (tested bo
 		return true, nil
 	}
 }
+
+func (admitter *SecretAdmitter) isOvaUrlChanged() (bool, error) {
+
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		log.Error(err, "Couldn't get the cluster configuration")
+		return false, err
+	}
+
+	err = api.SchemeBuilder.AddToScheme(scheme.Scheme)
+	if err != nil {
+		log.Error(err, "Couldn't build the scheme")
+		return false, err
+	}
+	err = apis.AddToScheme(scheme.Scheme)
+	if err != nil {
+		log.Error(err, "Couldn't add forklift API to the scheme")
+		return false, err
+	}
+
+	cl, err := client.New(config, client.Options{Scheme: scheme.Scheme})
+	if err != nil {
+		log.Error(err, "Couldn't create a cluster client")
+		return false, err
+	}
+
+	oldSecret := core.Secret{}
+	err = cl.Get(context.TODO(), client.ObjectKey{Namespace: admitter.secret.Namespace, Name: admitter.secret.Name}, &oldSecret)
+	if err != nil {
+		log.Error(err, "Couldn't get the target provider secrete")
+		return false, err
+	}
+
+	url := oldSecret.Data["url"]
+	newURL := admitter.secret.Data["url"]
+	if string(url) != string(newURL) {
```

Comment:
converting byte arrays to strings just for the comparison is redundant, how about replacing this and the lines below with:
```golang
return !bytes.Equal(url, newURL), nil
```